sdk/appconfiguration/azure-appconfiguration/tests/perfstress_tests/get_set.py

```diff
@@ -0,0 +1,42 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+
+from azure_devtools.perfstress_tests import PerfStressTest
+
+from azure.appconfiguration import ConfigurationSetting, AzureAppConfigurationClient as SyncAppConfigClient
+from azure.appconfiguration.aio import AzureAppConfigurationClient as AsyncAppConfigClient
+
+
+class GetSetTest(PerfStressTest):
+    service_client = None
+    async_service_client = None
+
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        connection_string = self.get_from_env("AZURE_APP_CONFIG_CONNECTION_STRING")
+        self.service_client = SyncAppConfigClient.from_connection_string(connection_string=connection_string)
+        self.async_service_client = AsyncAppConfigClient.from_connection_string(connection_string=connection_string)
```

Comment:
If you're planning to add a suite of tests for AppConfig, you can move this into a test base (like I linked to for ServiceBus above). However if you're only planning 1-2 tests, then it's probably simpler to keep as-is.